SUPPORT.md

```diff
@@ -0,0 +1,22 @@
+<!-- https://help.github.com/en/github/building-a-strong-community/adding-support-resources-to-your-project -->
+
+# Support
+
+Having trouble? Please consult these official resources in order of appearance.
+
+## Resources
+
+**[open.inf.is][docs-site] on the Web:** The OpenINF website. It has docs and
+tutorial resources.
+
+**[`#openinf`][matrix-channel] on matrix.org:** The OpenINF Matrix support channel
+on the _matrix.org_ network. It’s a synchronous chat channel for lightweight
+communication. Good uses of it are for asking questions, casual conversation,
+collaboration, and quick exchanges.
+
+**[@openinf][twitter-account] on Twitter:** The OpenINF support account on
+Twitter.
+
+[docs-site]: https://open.inf.is/docs
+[matrix-channel]: https://matrix.to/#/#openinf:matrix.org
+[twitter-account]: https://twitter.com/OpenINF
```

Comment:
```suggestion
<!-- prettier-ignore-start -->
<!-- PRESERVE LINK DEFINITION LABEL CASE - START -->

[Twitter Rules]:
  https://help.twitter.com/en/rules-and-policies/twitter-rules
  'The Twitter rules: safety, privacy, authenticity, and more'

[Code of Conduct]:
  ./CODE_OF_CONDUCT.md
  'Standards for how to engage with the project community'

<!-- PRESERVE LINK DEFINITION LABEL CASE - END -->
<!-- prettier-ignore-end -->
```

Reply:
suggest, use markdownlint toolchain

## library

see https://github.com/DavidAnson/markdownlint

## cli tool

for gh actions:

```yaml
- name: Run markdown checking
  run: npx markdownlint-cli2 "**/*.md"
```

see https://github.com/DavidAnson/markdownlint-cli2

## vscode extension

see https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint